zboss/doc/zboss_configuration.rst

```diff
@@ -5,9 +5,26 @@ ZBOSS configuration
 
 The ZBOSS stack is distributed as a set of precompiled linkable libraries.
 There are two variants of the libraries that support different Zigbee device roles, each variant having its own configuration.
-For a complete list of the ZBOSS configuration options, see the following files:
+For a complete list of the ZBOSS configuration options, used for building the production-ready version, see the following files:
 
-* :file:`zboss/include/osif/libzboss_config.h` - library for Coordinators and Routers
-* :file:`zboss/include/osif/libzboss_config.ed.h` - library for End Devices
+* :file:`zboss/v3.3/include/osif/libzboss_config.h` - library for Coordinators and Routers
+* :file:`zboss/v3.3/include/osif/libzboss_config.ed.h` - library for End Devices
+
+For a complete list of the ZBOSS configuration options, used for building the experimental (NCP-enabled) version of ZBOSS, see the following files:
+
+* :file:`zboss/v3.99/include/osif/libzboss_config.h` - library for Coordinators and Routers
+* :file:`zboss/v3.99/include/osif/libzboss_config.ed.h` - library for End Devices
```

Comment:
What does 3.99 stand for? I thought ZBOSS version for NCP was 3.4.0.7?

Reply:
The idea was to have a generic name for "the newest ZBOSS from v3 branch".

Reply:
Naming convention updated to the `production/development` pair.